팩토리얼.js

```diff
@@ -0,0 +1,10 @@
+let fs = require('fs');
+let input = fs.readFileSync('/dev/stdin').toString().split(' ');
+
+let n = Number(input);
```

Comment:
```suggestion
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split(' ');

const n = Number(input);
```
n === 1인 경우는 해결이되는지 궁금하네요